django/db/migrations/serializer.py

```diff
@@ -320,6 +321,11 @@ def serialize(self):
                 return "%s.%s" % (module, self.value.__name__), {"import %s" % module}
 
 
+class UUIDSerializer(BaseSerializer):
+    def serialize(self):
+        return "uuid.%s" % repr(self.value), {"import uuid"}
```

Comment:
Other than in error messages, the rest of this file calls `repr()` explicitly, FWIW.